src/components/WebexMeetingGuestAuthentication/WebexMeetingGuestAuthentication.jsx

```diff
@@ -1,25 +1,29 @@
-import React, {useState, useContext} from 'react';
+import React, {useState, useContext, useRef} from 'react';
 import PropTypes from 'prop-types';
 import webexComponentClasses from '../helpers';
-import {Button} from '../generic';
+import {Button, Icon} from '../generic';
 import {PasswordInput, TextInput} from '../inputs';
 import {PHONE_LARGE} from '../breakpoints';
-import {useElementDimensions, useMeeting, useRef} from '../hooks';
+import {useElementDimensions, useMeeting} from '../hooks';
 import {AdapterContext} from '../hooks/contexts';
 import Spinner from '../generic/Spinner/Spinner';
+import CaptchaInput from '../inputs/CaptchaInput/CaptchaInput';
 
 const HINTS = {
```

Comment:
Why is this called HINTS? Could we find a better name?

Reply:
This is not something that we changed. @sreenara if you want we can change this name - I want to point out that these are the translations that will be moved to a new file and there we will be changing the name. So this change will come in the scope of this JIRA - https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-538547